### ISSUE TITLE
Add GPS support

### ISSUE THREAD

**Issue by derchrome** [cjmoran/cgeo-wear#18]
Hi guys,
I would like to use the watches GPS for navigating (curiously it is more accurate then the phones one). Is there a chance to implement that?

Besides that I have some ideas to improve the app (e.g. store some caches directly on the watch, etc) and also donloaded the zip file but unfortunately I am not able to build it with android studio :-(
Maybe someone could help me getting started?
Is there any chance to send private messages in github?

**Comment by Lineflyer** [comment]
Hi @derchrome.
I am not sure if this repo is still maintained. There was no change over a longer time.
Maybe @cjmoran can tell more?!

I would also prefer if this nice project is continued and updated with more functionality.

**Comment by derchrome** [comment]
Thanks @Lineflyer. I would like to work on it. But as mentioned before I could not get started with this project in Android studio. Could you be so kind and help me getting started?

**Comment by cjmoran** [comment]
Yeah, I don't maintain this anymore, sorry. I tried to patch it once a few months ago but I was also having trouble getting it running in Android Studio so I gave up.

Anyway, watch GPS support is already in the app, though I have no idea whether it works with watches newer than the 1st gen of Android Wear. You can try it by enabling it in the phone-side app.

**Comment by derchrome** [comment]
In the phone side app there is no option to enable GPS. If I turn if the GPS on my mobile the app doesn't get the position. I would really like to develop on it but if no one knows how to get started :-(

How did you develop on it in the past? Also in Android studio?

**Comment by cjmoran** [comment]
You don't see this option? http://i.imgur.com/3uPyuog.png

This was originally developed in Android Studio, but that was a while ago so it would be a very old version at this point. Theoretically you should be able to get it building in any environment, but I'm unfortunately not familiar enough with the current Android build process to give any guidance. It would be a bit of an undertaking for me and I've got other stuff on my plate right now :(

I think it would probably be better if someone just re-made this from the ground up. It's not too complex an app and you can take whatever code snippets you want from my repo. I'm not even sure if I implemented the compass thing correctly because it never really worked right on my watch. I was never able to determine if it was a hardware problem or a programming error on my part; other users have reported the app working well but I have no way of knowing whether it works because my only AW watch is this OG Moto 360 with potentially buggy hardware.
